### DISCUSSION TITLE
Terminate sector

### DISCUSSION THREAD

**Discussion by s0nik42** [filecoin-project/community#375] [Network + Implementations Announcements]
Could the terminate sector be bind only to the Owner address to protect owner from seeing all their sectors terminated ?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
https://filecoinproject.slack.com/archives/C029MT4PQB1/p1633093358116500?thread_ts=1632999156.110000&cid=C029MT4PQB1
